src/css/buttons.css

```
@@ -0,0 +1,41 @@
+@layer utilities {
+    .button-base {
+        @apply relative border-none text-base leading-5 font-bold rounded-2xl w-full py-4 px-7 max-h-13 cursor-pointer gap-3 disabled:cursor-not-allowed disabled:pointer-events-none flex justify-center items-center flex-row
+    }
+
+    .button-primary {
+        @apply bg-theme-primary-700 dark:bg-theme-primary-650 text-white hover:bg-theme-primary-600 disabled:text-theme-secondary-400 disabled:bg-theme-secondary-100 dark:disabled:bg-theme-secondary-700 focus:shadow-[0_0_0_4px_#E5F3ED] dark:focus:shadow-[0_0_0_4px_rgba(74,205,151,0.24)] focus-visible:outline-2  focus-visible:outline-theme-primary-600
+    }
+
+    .button-secondary {
+        @apply text-theme-primary-700 dark:text-theme-primary-650 bg-white dark:bg-light-black border border-solid border-theme-primary-700 dark:border-theme-primary-650 hover:bg-theme-primary-50 dark:hover:bg-theme-secondary-800 focus-visible:outline-none focus-visible:shadow-[0_0_0_4px_#E5F3ED] dark:focus-visible:shadow-[0_0_0_4px_#2B3D35] disabled:border disabled:border-solid disabled:border-theme-secondary-100 dark:disabled:border-theme-secondary-400 disabled:text-theme-secondary-500 dark:disabled:text-theme-secondary-300 disabled:bg-white dark:disabled:bg-light-black
+    }
```

maybe try to split this up to make it more readable, e.g.

```suggestion
    .button-secondary {
        @apply text-theme-primary-700  bg-white border border-solid border-theme-primary-700 dark:text-theme-primary-650 dark:bg-light-black dark:border-theme-primary-650;
        @apply hover:bg-theme-primary-50 dark:hover:bg-theme-secondary-800;
        @apply focus-visible:outline-none focus-visible:shadow-[0_0_0_4px_#E5F3ED] dark:focus-visible:shadow-[0_0_0_4px_#2B3D35];
        @apply disabled:border disabled:border-solid disabled:border-theme-secondary-100 dark:disabled:border-theme-secondary-400 disabled:text-theme-secondary-500 dark:disabled:text-theme-secondary-300 disabled:bg-white dark:disabled:bg-light-black;
    }
```

it's a lot of css cramped together otherwise. Likewise for other definitions